lib/controller.js

```diff
@@ -39,13 +39,16 @@ function lnCmsController($rootScope, $scope, lnCmsClientService, $urlRouter) {
         //load view
         lnCmsClientService.getView(endpoint, params)
           .then(function(response) {
-            if (response.data.length > 0)
+            if (response.data.length > 0) {
```

Comment:
In that case will follow under the else if.

> On Apr 4, 2016, at 9:39 PM, Adam Fenton <notifications@github.com> wrote:
> 
> In lib/controller.js <https://github.com/moxie-leean/ng-cms/pull/6#discussion_r58479771>:
> 
> > @@ -39,13 +39,16 @@ function lnCmsController($rootScope, $scope, lnCmsClientService, $urlRouter) {
> >          //load view
> >          lnCmsClientService.getView(endpoint, params)
> >            .then(function(response) {
> > -            if (response.data.length > 0)
> > +            if (response.data.length > 0) {
> It's not from staging - http://wp.skaled.moxie-staging.com/wp-json/leean/v1/post?id=1 <http://wp.skaled.moxie-staging.com/wp-json/leean/v1/post?id=1> returns an object:
> http://snaps.getmoxied.net/3v2z0T1F0n2L <http://snaps.getmoxied.net/3v2z0T1F0n2L>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/moxie-leean/ng-cms/pull/6/files/bfcc4884b2934af2d9d2f11fa483402048416224#r58479771>